version_checker.install

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Contains version_checker.module.
+ */
+
+use Drupal\user\Entity\Role;
+
+function version_checker_install() {
+  foreach(['anonymous', 'authenticated'] as $role) {
```

Comment:
Ik zou een var maken van de array met allowed roles, mochten hier dingen bij of af moeten is het wat inzichtelijker.
Ik snap wel dat dit efficienter is